Lib/unittest/case.py

```diff
@@ -1057,9 +1057,11 @@ def assertSequenceEqual(self, seq1, seq2, msg=None, seq_type=None):
                     differing += ('Unable to index element %d '
                                   'of second %s\n' % (len1, seq_type_name))
         standardMsg = differing
-        diffMsg = '\n' + '\n'.join(
-            difflib.ndiff(pprint.pformat(seq1).splitlines(),
-                          pprint.pformat(seq2).splitlines()))
+        diffMsg = difflib.unified_diff(pprint.pformat(seq1).splitlines(),
+                                        pprint.pformat(seq2).splitlines(),
```

Comment:
align the hanging indents by removing a space.

Reply:
Solve it! Thanks!